doc/contributing/sharing-project-news.md

```diff
@@ -0,0 +1,30 @@
+# Sharing project news
+
+The project, supported by the next-10 team, aims to make it easy
+to share project news with those who publish newsletters
+that cover Node.js.
+
+The project's approach is as follows:
+
+* The project will create one "What's new in YYYY" issue in the `nodejs/node`
```

Comment:
An issue seems like the wrong tool for the job. Would a GitHub discussion item be better?